packages/perseus/src/widgets/numeric-input.jsx

```diff
@@ -421,8 +421,7 @@ export class NumericInput extends React.Component<Props, State> {
         const rubric = this.props.reviewModeRubric;
         const answers = this.getAnswerBlurb(rubric);
         const answerBlurb = answers[0];
-        const correct = answers[1];
-        const classes = this.getClasses(correct, rubric);
```

Comment:
Why did this change?

Reply:
The Wonderblock form element handles all of this now. These variables were used to calculate style and I don't need that now.